packages/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: 3ca6ba66032a1efce0500f9ad6f83351ad0604b8
-packageVersion: 1
+packageVersion: 2
```

Comment:
No need to bump the packageVersion; we only bump once per release and 14.5.101 has not been release yet